src/components/Footer/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+
+import Logo from "../Logo";
+import P from "../P";
+import Icon from "../Icon";
+import Row from "../Row";
+import Column from "../Column";
+import Mail from "../../assets/img/icons/mail.svg";
+import {
+  FooterWrapper,
+  Wrapper,
+  ContactWrapper,
+  FooterP,
+  MailA,
+} from "./styledComponents";
+import { white, copyrights } from "../../assets/styles/colors";
+
+const Footer: FC = () => {
+  return (
+    <FooterWrapper>
+      <Wrapper>
+        <Logo isBackgroundDark width="137px" />
+        <ContactWrapper>
+          <FooterP>Kontakt</FooterP>
```

Comment:
I think there is 4px padding missing between Kontakt and email address.

Reply:
Done